packages/sanity/src/_internal/cli/actions/deploy/deployAction.ts

```diff
@@ -143,6 +144,14 @@ export default async function deployStudioAction(
 
     // And let the user know we're done
     output.print(`\nSuccess! Studio deployed to ${chalk.cyan(location)}`)
+
+    if (!configStudioHost) {
+      output.print(
+        `\nPlease remember to add ${chalk.cyan(`studioHost: '${userApplication.appHost}'`)}`,
+      )
+      output.print('to defineCliConfig in your sanity.cli.js or sanity.cli.ts file')
+      output.print('to re-deploy or undeploy this studio in the future.')
```

Comment:
```suggestion
    if (!configStudioHost) {
      output.print(
        `\nAdd ${chalk.cyan(`studioHost: '${userApplication.appHost}'`)}`,
      )
      output.print('to defineCliConfig root properties in sanity.cli.js or sanity.cli.ts')
      output.print('to avoid prompting for hostname on next deploy.')
```

Reply:
This might be a little misleading since this message will _also_ turn up for legacy users that DO have a "default" project, and they won't receive a prompt. I'm okay with that, but wanted to ensure you were.